### PR TITLE
407 meta data

### PR DESCRIPTION
**Pull request by kordano**
#### SUMMARY
Adds meta data to DB, adds simplified schema functions to api.
See [discussion](https://github.com/replikativ/datahike/discussions/407)

##### Feature
- [x] Implements an existing feature request. Make sure the feature request has been accepted for implementation before opening a PR.
- [x] Related issues linked using `fixes #number`
- [x] Integration tests added
- [x] Documentation added
- [x] Formatting checked

### CODE REVIEW

File: src/datahike/schema.cljc [hunk 1]
```diff
@@ -37,7 +37,11 @@
     :db.type/symbol
     :db.type/uuid
     :db.type/value
-    :db.type/tuple})
+    :db.type/tuple
+    :db.type/cardinality
+    :db.type.install/attribute
+    :db.type/valueType
+    :db.type/unique})
```
Comment (jsmassa) [inline src/datahike/schema.cljc:44]: Why did you add these?

Comment (kordano) [reply src/datahike/schema.cljc:44]: These are the allowed types for system attributes. `:db/cardinality` for instance has the allowed values `:db.cardinality/many` and `:db.cardinality/one` as type `.db.type/cardinality`.

Comment (jsmassa) [reply src/datahike/schema.cljc:44]: I see...

**Review by jsmassa — APPROVED** [review]
Everything addressed now
